keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/annealing/ResourceActuator.kt

```diff
@@ -4,35 +4,41 @@ import com.netflix.spinnaker.keel.api.ApiVersion
 import com.netflix.spinnaker.keel.api.Resource
 import com.netflix.spinnaker.keel.api.ResourceName
 import com.netflix.spinnaker.keel.persistence.ResourceRepository
-import com.netflix.spinnaker.keel.persistence.ResourceState.Diff as Different
+import com.netflix.spinnaker.keel.persistence.ResourceState.Diff
```

Comment:
WTF Cam?

Reply:
loool I was building you a DifferenceEngine

Reply:
actually I created a class originally called Diff that I subsequently renamed but I forgot where I did this thing sorry

Reply:
I figured you were importing something from the diff lib at one point, yeah.